nationality-park/park.js

```diff
@@ -0,0 +1,50 @@
+import parksData from "./nationalParksOfPolandData.json" assert { type: "json" };
+
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString);
+const name = urlParams.get("parkname");
+
+if (name == null) {
+  window.location.href = "index.html";
+}
+window.addEventListener("DOMContentLoaded", () => {
```

Comment:
move it to the end of the file, be consist with the same patterns, do same things in the same way everywhere